src/views/Workflow.vue

```diff
@@ -17,24 +17,47 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 <template>
   <div id="workflow-panel" class="fill-height">
-    <workflow
-      :workflow-name="workflowName"
-      :workflow-tree="tree"
-      :is-loading="isLoading"
-      ref="workflow-component" />
+    <lumino
+      ref="lumino"
+      v-on:lumino:deleted="onWidgetDeletedEvent"
+    >
+      <tree-component
+        v-for="widgetId of treeWidgets"
+        :key="widgetId"
+        :id="widgetId"
+        :workflows="tree.root.children"
+      />
+      <graph-component
+          v-for="widgetId of graphWidgets"
+          :key="widgetId"
+          :id="widgetId"
+          :workflow-name="workflowName"
+      />
+      <mutations-view
+          v-for="widgetId of mutationsWidgets"
+          :key="widgetId"
+          :id="widgetId"
+          :workflow-name="workflowName"
+      />
+    </lumino>
```

Comment:
We could go even further and remove the need to have computed properties for the widgets. So that if a new component type is added, we don't have to modify anything in the code.

But that's for another PR. This PR is only about the multiple wrappers (IOW there's room for improvement in this view, but at least the mess of wrappers should be fixed :+1: )